app/front/public/gallery.json

```diff
@@ -647,6 +647,7 @@
 {"name":"Seti","url":"https://raw.githubusercontent.com/ctf0/Seti_UX/master/Seti.tmTheme","light":false},
 {"name":"Shipwreck","url":"https://raw.githubusercontent.com/squidfunk/textmate-themes/master/Shipwreck.tmTheme","light":false},
 {"name":"Shrek","url":"https://raw.githubusercontent.com/daylerees/colour-schemes/master/sublime/shrek.tmTheme","light":false},
+{"name":"Sky Blue","url":"https://github.com/MikoSky01/Scheme-For-Sublime/blob/master/Scheme/themes/Blue%20Sky.tmTheme":false},
```

Comment:
Missing Key before false

Reply:
also link seems broken

Reply:
Guees you meant [this link](https://github.com/MikoSky01/Sublime-theme/blob/master/themes/Blue%20Sky.tmTheme)

Reply:
Okay